scopes/workspace/workspace/workspace.ts

```diff
@@ -1277,6 +1280,73 @@ export class Workspace implements ComponentFactory {
       await this.importObjects();
       this.logger.consoleSuccess();
     }
+    return this.installModules(options);
+  }
+
+  private async getComponentsWithDependencyPolicies() {
+    const allComponentIds = await this.listIds();
+    const componentConfigFiles: Record<string, ComponentConfigFile> = {};
+    const componentPoliciesById: Record<string, any> = {};
+    (
+      await Promise.all<ComponentConfigFile | undefined>(
+        allComponentIds.map((componentId) => this.componentConfigFile(componentId))
+      )
+    ).forEach((component) => {
+      if (component == null) return;
+      const depResolverAspect = component?.aspects.get(DependencyResolverAspect.id);
+      if (depResolverAspect == null || component == null) return;
```

Comment:
why do you need here the `|| component == null` ? you already have it few lines above.
Also why not using `if (!depResolverAspect)` instead of `== null`

Reply:
`component == null` is not needed. 

`depResolverAspect == null` is the same as `!depResolverAspect` in this case. I can change it if it is more preferred. Sometimes there are some edge cases with `!foo` instead of `foo == null`. Like `!foo` would be true on an empty string. So there are linting rules that suggest to use `== null` instead. That's why I am used to it.